rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
- [ ] Kindly make sure to include `book` and `person` in the attr_accessor so they have getters and setters setup

```suggestion
  attr_accessor :date, :book, :person
```